.gitignore

```diff
@@ -1,34 +1,25 @@
-node_modules/
-artifacts/
-**/gen/**/build/
-vendor/
-_local/
-*.swp
-*.swo
+# Hidden files
+.*
```

Comment:
This is a bit confusing. In the commit msg it says "all hidden files and directories", but we have some directories that we don't want to ignore like `.github, .gitmodules, .gitignore` and probably a couple of others. I'm not sure if we want to ignore all the hidden files & directories. Besides we double ignore dirs like: `.vscode` or `.idea` if we ignore all hidden.

Reply:
Good point! Normally I'd just ignore all dot-prefixed files and exclude the ones we don't want to be ignored:
```
.*
!.gitignore
!.dockerignore
```

But taking into account the number of dot-prefixed files we want to commit it will be easier to list the ones we want to ignore.

Reply:
e50512f594158c611537327df812df08cef86e94